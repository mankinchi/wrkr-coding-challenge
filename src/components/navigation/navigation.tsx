import { NavigationItem } from "./components/navigationItem";

export const Navigation = () => (
	<div className="flex flex-col">
		<NavigationItem
			title="Home"
			path="/"
		/>
		<NavigationItem
			title="Search"
			path="/search"
		/>
		<NavigationItem
			title="Showcase"
			path="/showcase"
		/>
	</div>
);
