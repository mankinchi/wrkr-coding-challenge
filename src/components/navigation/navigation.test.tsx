import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import { Navigation } from "./navigation";

describe("Navigation", () => {
	it("should render correctly", () => {
		const { container } = render(
			<MemoryRouter initialEntries={["/"]}>
				<Navigation />
			</MemoryRouter>,
		);

		expect(container).toMatchSnapshot();
	});
});
