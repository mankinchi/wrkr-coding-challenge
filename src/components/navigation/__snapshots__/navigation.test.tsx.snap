// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render correctly 1`] = `
<div>
  <div
    class="flex flex-col"
  >
    <a
      href="/"
    >
      <div
        class="cursor-pointer border-b border-slate-600 px-2 py-4 transition-all hover:font-bold font-semibold"
      >
        Home
      </div>
    </a>
    <a
      href="/search"
    >
      <div
        class="cursor-pointer border-b border-slate-600 px-2 py-4 transition-all hover:font-bold"
      >
        Search
      </div>
    </a>
    <a
      href="/showcase"
    >
      <div
        class="cursor-pointer border-b border-slate-600 px-2 py-4 transition-all hover:font-bold"
      >
        Showcase
      </div>
    </a>
  </div>
</div>
`;
